coderd/database/queries/notificationsinbox.sql

```diff
@@ -0,0 +1,51 @@
+-- name: GetInboxNotificationsByUserID :many
+SELECT * FROM inbox_notifications WHERE
+	user_id = @user_id AND
+	(@read_status::text = 'ALL' OR (@read_status::text = 'UNREAD' AND read_at IS NULL) OR (@read_status::text = 'READ' AND read_at IS NOT NULL)) AND
+	(@created_at_opt::TIMESTAMPTZ = '0001-01-01 00:00:00Z' OR created_at < @created_at_opt::TIMESTAMPTZ)
+	ORDER BY created_at DESC
+	LIMIT (COALESCE(NULLIF(@limit_opt :: INT, 0), 25));
+
+-- name: GetInboxNotificationsByUserIDFilteredByTemplatesAndTargets :many
```

Comment:
Nit: for future-proofing, rather drop the specificity here in case we add more filter options later:

```suggestion
-- name: GetFilteredInboxNotificationsByUserID :many
```

Reply:
Renamed ✅